content/03-reference/01-tools-and-interfaces/02-prisma-client/057-null-and-undefined.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Null and undefined'
+metaTitle: 'Null and undefined (Reference)'
+metaDescription: 'How Prisma Client handles null and undefined, including a GraphQL use case.'
+preview: false
+---
+
+<TopBlock>
+
+Prisma Client differentiates between `null` and `undefined`:
+
+* `null` is a **value**
+* `undefined` means **do nothing**
+
+> **Note**: This is particularly important to account for in [a **Prisma with GraphQL context**, where `null` and `undefined` are interchangeable](#use-case-null-and-undefined-in-a-graphql-resolver).
+
+In the following example, if `emailInput` is `null`, the query sets `email` (a **mandatory** field) to `undefined` - which means ✔ **do not include this in the update**:
+
+```ts
+const update = await prisma.user.update({
+  where: {
+    id: 1,
+  },
+  data: {
+    name: "Petunia",
+|  email: emailInput != null ? emailInput : undefined, // If null, don't include in update!
```

Comment:
Is that `|` intentional?

Reply:
It is - to highlight the line mentioned in the intro. Does it look odd? 🤔

Reply:
No it makes sense. I just wasn't aware of this line highlighting feature :)

Reply:
🙏